### ISSUE TITLE
Nanopub retractions should allow leaving a comment

### ISSUE THREAD

**Issue by Ostrzyciel** [knowledgepixels/nanodash#185]
I'm not sure where to post an issue about this, but it seems that the retraction template is at least partially hardcoded into Nanodash, so I'm putting it here.

When I post a retraction in Nanodash using [this template](https://nanodash.knowledgepixels.com/explore?id=RA0QOsYNphQCityVcDIJEuldhhuJOX3GlBLw6QylRBhEI) I often would like to specify the reason for making the retraction. I don't know if this reason specification should be structured in some manner – for me, just `rdfs:comment` would be enough. Currently there is no option to do this.

**Comment by tkuhn** [comment]
Yes, makes a lot of sense. I see how such a comment would be useful.

I was at some point considering something similar for superseding nanopubs, which would be a bit like a commit message for an update in a nanopub.

rdfs:comment seems a good match too.

I will look into this. Should be straightforward.

**Comment by tkuhn** [comment]
Retraction template is updated so it allows to leave a comment: https://nanodash.knowledgepixels.com/publish?template=http://purl.org/np/RA_fYtKwFVA-fVS72uKV77LDucqrpNn0cbICKicCm-0wM
